.github/workflows/release.yml

```diff
@@ -6,7 +6,17 @@ on:
       dryrun:
         type: boolean
         description: 'dryrun: run without pushing SCM changes to upstream'
-        default: true
+        default: false
+
+      skip-tests:
+        type: boolean
+        description: 'skip-tests: do not run tests while releasing'
+        default: false
+
+      target-tag:
+        type: string
+        description: 'target-tag: tag or branch name to release. Use to to re-release failed releases'
```

Comment:
I don't know why it's set that way, but from what I see it will target 1 commit before the branch if it's not master. May be worth including that in the description, because it may be not obvious to the user.